src/routes/v2/organisations.js

```diff
@@ -78,7 +83,7 @@ module.exports = [
     },
     {
         method: 'POST',
-        path: '/v2/organisations/{orgUuid}/add',
+        path: '/v2/organisations/{orgUuid}/addUser',
```

Comment:
I believe normally we'd just use verbs for the rest api to describe what happens, and the POST is for add/create, then just use a verb-less url such as  /users on the end instead of /addUser